Adding e2e tests for run_playbook action

ttuffin opened the pull request:
Closes [AAP-8660](https://issues.redhat.com/browse/AAP-8660).

Alex-Izquierdo commented on tests/e2e/files/playbooks/post_processing.yml:
```diff
@@ -0,0 +1,51 @@
+---
```
The name of this file is confusing. This playbook is for a specific testcase, is not for general purposes as "print_event". The name should include what test belongs to.

Alex-Izquierdo commented on tests/e2e/files/rulebooks/actions/test_run_playbook.yml:
```diff
@@ -0,0 +1,56 @@
+  hosts: all
+  sources:
+    - generic:
+        shutdown_after: 6
```
Can you please include this value as envvar injected in the test? This would be a preparation for: https://issues.redhat.com/browse/AAP-9047

Alex-Izquierdo commented on tests/e2e/test_actions.py:
```diff
@@ -94,3 +95,48 @@ def test_actions_sanity():
+    """
+
+    rulebook = utils.BASE_DATA_PATH / "rulebooks/actions/test_run_playbook.yml"
+    os.environ["SOURCE_SHUTDOWN_AFTER"] = "6"
```
This set the environment variable for the main process, generating potential side effects. You must create a copy of the environment, update it and inject it only for this specific command. Example: https://github.com/ansible/event-driven-ansible/blob/main/tests/integration/event_source_webhook/test_webhook_source.py#L44

ttuffin replied:
@Alex-Izquierdo done. Pls re-review

ttuffin commented:
@Alex-Izquierdo please re-review. I have updated the test to use the new env fixture and added a new commit to include `pytest-check` and updated the test.

mkanoor commented on tests/e2e/files/playbooks/run_playbook_test_playbook.yml:
```diff
@@ -0,0 +1,51 @@
+      when: remediation_required
+      block:
+        - name: Check if temp file exists
+          ansible.builtin.stat:
```
@ttuffin We have seen that for some reason the stat call on localhost causes the ansible-playbook to do an auto discovery of python. And the results are different on Mac and Linux.
We were able to fix this in PR #365 by adding the python interpreter in the inventory.yml

ttuffin replied:
Thanks @mkanoor, thats correct, we are checking for strings in stdout so it shouldn't be affected. Will keep it in mind though.
